src/app/components/mnoe-organizations-list/mnoe-organizations-list.coffee

```diff
@@ -96,7 +102,6 @@
 
     displayCurrentState = () ->
       setAllOrganizationsList()
```

Comment:
The only problem I can see happening is when an organization is updated. There is a callback registered on line 139 that will be triggered when an organization is updated, and it is reliant on this call to `fetchOrganizations`. Perhaps we could move this call to the `onOrganizationChanged` callback? So then the extra call to `fetchOrganizations` will only occur when necessary (when an org is changed/updated).

Reply:
@adamaziz15 I see what you mean. I've re-added the `fetchOrganizations` here since yes this method does effect the `onOrganizationChanged` and was also effecting the `searchMode`.

Changed the initial directive call instead to just set the text.